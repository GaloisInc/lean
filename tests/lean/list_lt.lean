#eval to_bool ([1, 2] < [2, 1])
#eval to_bool ([2, 1] < [1, 2])
#eval to_bool ("ab" < "ba")
#eval to_bool ("ba" < "ab")
#reduce to_bool ("ba" < "ab")
example : Â¬ ("ba" < "ab") := dec_trivial
