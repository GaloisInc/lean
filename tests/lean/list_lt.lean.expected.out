tt
ff
tt
ff
ff
